:sparkles: Read/Write Verilog to/from AIGs (#128)
marcelwa/aigverse: README.md

Additionally, you can read AIGER files into sequential AIGs using `read_aiger_into_sequential_aig` and
`read_ascii_aiger_into_sequential_aig`.

> You can read a verilog file as an AIG.

Reviewer: ```suggestion
You can read a Verilog file as an AIG.
```

Author: Done.

> You can read a verilog file as an AIG.

Reviewer: As mentioned in the review for #127, it is best to open one section per file type or merge them into one section with one prominent example for reading and writing. Please, add a link to a reference for the gate-level Verilog support that `mockturtle` offers.

Author: Done.
Btw, the latest information I could see in lorina's PR indicates it based on simple subset of Verilog-95, but I'm not sure the support of the current version which is based on regular expression. I linked the Verilog keywords to 95 standard. I could remove it if you feel improper.

Author: I created a separate section in README.md but I didn't create a new docs/verilog.md for verilog since based on my understanding Verilog won't be worked as a main internal structure on optimisation in mockturtle, there won't be too much necessary to expose, but indeed a good/readable I/O format for users. 
If you still think it's necessary to create one, please let me know.

```python
from aigverse import read_verilog_into_aig

# Read verilog files into AIG networks

> aig = read_aiger_into_aig("example.v")

Reviewer: ```suggestion
aig = read_verilog_into_aig("example.v")
```

Author: Done.

# Print the size of the AIGs
print(f"AIG Size: {aig.size()}")
```

#### Writing

```python

> from aigverse import write_aiger

Reviewer: ```suggestion
from aigverse import write_aiger, write_verilog
```

Author: Done.

# Write an AIG network to an AIGER file
write_aiger(aig, "example.aig")

> # Write an AIG network to a verilog file

Reviewer: ```suggestion
# Write an AIG network to a Verilog file
```

Author: Done.

> write_aiger(aig, "example.v")

Author: Done. 
Indeed a real typo.
This is the first time I see copilot reviewing, looks amazing!

write_aiger(aig, "example.aig")
```

> ### Verilog Files
> 
> You can read and write [Verilog](https://ieeexplore.ieee.org/document/803556) files.

Reviewer: I understand there is no direct reference for the gate-level subset of Verilog that `lorina` and `mockturtle` (and hence `aigverse`) support. I appreciate your effort in linking to this original Verilog specification. However, I think it might be misleading since we support only such a small subset. Let's rather say something like the following then:
```suggestion
### Verilog Files

You can read and write gate-level Verilog files, which constitutes a very small subset of the Verilog standard, similar in extent to what ABC supports. For more information, see the [`lorina` parser](https://lorina.readthedocs.io/en/latest/verilog.html) used by this project.
```

> #### Parsing
> 
> You can read a Verilog file as an AIG.
> 

Reviewer: ```suggestion
#### Parsing

```

Reviewer: For consistency with AIGER

Author: Done.

```python
from aigverse import read_verilog_into_aig

> # Read Verilog files into AIG networks
> aig = read_verilog_into_aig("example.v")

Reviewer: ```suggestion
# Read a Verilog file into an AIG network
aig = read_verilog_into_aig("example.v")
```

Author: Done.

> # Print the size of the AIGs

Reviewer: ```suggestion
# Print the size of the AIG
```

Author: Done.

print(f"AIG Size: {aig.size()}")
```

> #### Writing
> 
> You can write an AIG to a Verilog file.
> 

Reviewer: ```suggestion
#### Writing

```

Reviewer: For consistency with AIGER
